src/Dependencies/Installer.php

```diff
@@ -96,4 +87,22 @@ private function installDependency($type, $dep)
 
         return \Result\Result::ok();
     }
+
+    private function inspectionDetailsMessage($type, $dep)
+    {
+        $result = $this->inspectionChecker->check($type, $dep);
+
+        if (!$result->isErr()) {
+            $inspectionDetails = $result->unwrap();
+            if (!empty($inspectionDetails)) {
+                $message = sprintf("%s\n", $inspectionDetails);
```

Comment:
Sure. At this point we have a variable being set and overriden, in theory multiple times (until you mentally parse that those two `if`s are mutually exclusive. This sort of situation leads to code which is hard to debug, since there are several places that a variable might be getting its value from. 

The nested conditional is bad, but at least it's clear that `$message` is getting its value from one of three places.